spec/trial_spec.rb

```diff
@@ -426,6 +426,59 @@ def expect_alternative(trial, alternative_name)
     end
   end
 
+  describe "#within_conversion_time_frame?" do
+    let(:trial) { Split::Trial.new(:user => user, :experiment => experiment) }
+
+    it "memoizes the result" do
+      allow(Split.configuration).to receive(:experiments).and_return(experiment.name => { "window_of_time_for_conversion_in_minutes" => 60 })
+      trial.choose!
+
+      expect(Time).to receive(:parse).once.and_call_original
+
+      trial.within_conversion_time_frame?
+      trial.within_conversion_time_frame?
```

Comment:
For my own knowledge, I was wondering why do we call the same function twice after the expect check?

Reply:
Yup that's right!